app/helpers/application_helper/toolbar/ems_middleware_center.rb

```diff
@@ -69,7 +69,9 @@ class ApplicationHelper::Toolbar::EmsMiddlewareCenter < ApplicationHelper::Toolb
         button(
           :ems_middleware_recheck_auth_status,
           'fa fa-search fa-lg',
-          N_("Re-check Authentication Status for this #{ui_lookup(:table=>'ems_middleware')}"),
+          proc do
+            _("Re-check Authentication Status for this %{provider}") % {:provider => ui_lookup(:table=>'ems_middleware')}
```

Comment:
`"Re-check Authentication Status for this Middleware Provider"`

Then the `proc do` can go away and be replaced with a plain `N_()`.